### PR TITLE
feat: Update docs, add new deps, fix bugs.

### LINKED ISSUE #67: Update README
The readme should be updated with the new changes that have been introduced with the v1.0.0.

This update should include:
- Whole refactoring of the README file.
- Add new scripts added in the `package.json` that influence the usage of the local server.
- Add new specifics:
  - Information about E2E tests
  - Add contacts of main contributors for users to have way to communicate with.

### LINKED ISSUE #68: Add scrolling behavior and page transition
It would be nice to add a nice smooth scroll behavior and page transition to the website.

For this to be completed we would need to:
- Either adding [Lenis](https://lenis.studiofreight.com/) for smooth scrolling which is native to the browser
- Add the way the router should react to a page change see [Vue Router](https://router.vuejs.org/guide/advanced/scroll-behavior.html).
- Add the page transitions with [Vue Router](https://router.vuejs.org/guide/advanced/transitions.html)

### LINKED ISSUE #69: Add UnoCSS dependency
As one the restyling of UI/UX is going to happen, we need to add another way to handle the CSS we utilize throughout the project. For that, I think the best way to do that would be to add [UnoCSS](https://unocss.dev/) as a dependency for the project as it combines the best qualities of [WindiCSS](https://windicss.org/), [TailwindCSS](https://tailwindcss.com/).

It has also been created by the [author](https://github.com/antfu) of [VueUse](https://vueuse.org/) and many other Vue related projects. So it would fit nicely into the project.

### LINKED ISSUE #74: Fix Service Worker Error
After the migration an error is occurring with the Service Worker. Figure out why it's breaking.

Image

### PR DESCRIPTION
**Pull request by Readpato**
## Changes

**What kind of change does this PR introduce?** (check at least one by adding an "x" between the brackets)

- [X] Bugfix
- [X] Feature
- [ ] Code style update
- [X] Refactor
- [ ] Build-related changes
- [X] Other, please describe:

Other:
- Update of the README file all the new required setup and information that needed to be added. Closes #67
- Add workspace settings for VSCode. 
- Install UnoCSS for styling. Closes #69
- Fix Service worker error. Closes #74
- Add scrolling behavior and page transition. Closes #68

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [X] No

If yes, please describe the impact and migration path for existing applications: -

**Screenshots and videos**

- If applicable, add screenshots or a video to help explain your problem.

- For more information on the supported file image/file types and the file size limits, please refer
to the following [link](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/attaching-files).

**The PR fulfills these requirements:**

- [X] Schrödinger Hat [code of conduct](https://www.schrodinger-hat.it/code-of-conduct)

### CODE REVIEW

**Review by AngyDev — APPROVED** [review]

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -2,16 +2,11 @@
 
 Current contributors:
 
+- [Angela Busato - AngyDev](https://github.com/https://github.com/AngyDev)
```
Comment (Wabri) [inline CONTRIBUTORS.md:5]:
```suggestion
- [Angela Busato - AngyDev](https://github.com/AngyDev)
```

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -2,16 +2,11 @@
...
 - [Gabriele Puliti - Wabri](https://github.com/Wabri)
-- [Miki Lombardi - TheJoin95](https://github.com/TheJoin95)
 - [Lorenzo Pieri - 404answernotfound](https://github.com/404answernotfound)
-- [Gianfabio Pezzolla - gianfabiopezzolla](https://github.com/gianfabiopezzolla)
+- [Miki Lombardi - TheJoin95](https://github.com/TheJoin95)
+- [Nicola Puppa - Nicpuppa](https://github.com/)
```
Comment (Wabri) [inline CONTRIBUTORS.md:9]:
```suggestion
- [Nicola Puppa - Nicpuppa](https://github.com/nicpuppa)
```

File: README.md [hunk 3]
```diff
@@ -129,9 +164,10 @@ Don't forget to give the project a star! Thanks again!
 
 ## Contact
 
-Schrodinger's Hat Team - [@schrodinger_hat](https://twitter.com/schrodinger_hat)
-
-Project Link: [https://github.com/Schrodinger-Hat/schrodinger-hat-website](https://github.com/Schrodinger-Hat/schrodinger-hat-website)
+- Team - [Organization repo](https://github.com/Schrodinger-Hat)
```
Comment (Wabri) [inline README.md:167]: Maybe better to redirect to people list of the org?
```suggestion
- Team - [Organization repo](https://github.com/orgs/Schrodinger-Hat/people)
```

**Comment by Readpato** [comment]
@Wabri Thank you for those reviews, totally oversaw those mistakes <3

**Pull request merged**